deploy/kairos/INSTALL.md

```diff
@@ -0,0 +1,113 @@
+# FUSE Installation Guide for Kairos HPC System
+
+## Overview
+This guide provides step-by-step instructions for installing the FUSE package and compiling it into a system image on the Kairos HPC server.
+
+## Requirements
+- Julia module available
+- GCC module available
+- Network access through proxy (203.230.124.39:4128)
+- Sufficient disk space (~10 GB per installation)
+
+## Environment Variables
+
+Configure the following environment variables before installation:
+
+| Variable | Description | Default Value |
+|----------|-------------|---------------|
+| `FUSE_HOME` | Base directory for FUSE installation | `/scratch/yoom/fuse` |
+| `FUSE_MODULE_DIR` | Directory for module files | `/scratch/yoom/fuse/modulefiles` |
```

Comment:
Update default values. Check that `yoom` does not occur anywhere else in these changes.